src/main/java/seedu/guest/logic/commands/AddCommand.java

```diff
@@ -29,14 +30,16 @@ public class AddCommand extends Command {
             + PREFIX_DATE_RANGE + "DATE_RANGE "
             + PREFIX_NUMBER_OF_GUESTS + "NUMBER_OF_GUESTS "
             + PREFIX_IS_ROOM_CLEAN + "IS_ROOM_CLEAN "
+            + PREFIX_REQUEST + "REQUEST(Optional) "
```

Comment:
Should we use square brackets to indicate optional? eg. "[" + PREFIX_REQUEST + "REQUEST]...\n" (This was how 'tag' was done in AB3).

Reply:
changed